packages/dds/tree/src/core/schema-stored/schema.ts

```diff
@@ -185,18 +214,41 @@ export class ObjectNodeStoredSchema extends TreeNodeStoredSchema {
 		super();
 	}
 
-	public override encode(): TreeNodeSchemaDataFormat {
+	public override encodeV1(): TreeNodeSchemaDataFormatV1 {
 		const fieldsObject: Record<string, FieldSchemaFormat> = Object.create(null);
 		// Sort fields to ensure output is identical for for equivalent schema (since field order is not considered significant).
 		// This makes comparing schema easier, and ensures chunk reuse for schema summaries isn't needlessly broken.
 		for (const key of [...this.objectNodeFields.keys()].sort()) {
+			const value = encodeFieldSchemaV1(
+				this.objectNodeFields.get(key) ?? fail(0xae6 /* missing field */),
+			);
+
 			Object.defineProperty(fieldsObject, key, {
 				enumerable: true,
 				configurable: true,
 				writable: true,
-				value: encodeFieldSchema(
-					this.objectNodeFields.get(key) ?? fail(0xae7 /* missing field */),
-				),
+				value,
+			});
+		}
+		return {
+			object: fieldsObject,
+		};
+	}
+
+	public override encodeV2(): TreeNodeSchemaDataFormatV2 {
+		const fieldsObject: Record<string, FieldSchemaFormat> = Object.create(null);
+		// Sort fields to ensure output is identical for for equivalent schema (since field order is not considered significant).
+		// This makes comparing schema easier, and ensures chunk reuse for schema summaries isn't needlessly broken.
+		for (const key of [...this.objectNodeFields.keys()].sort()) {
+			const value = encodeFieldSchemaV1(
+				this.objectNodeFields.get(key) ?? fail(0xae6 /* missing field */),
```

Comment:
We shouldn't duplicate assert/fail hex codes. The release process will take care of replacing the string with a new one.
```suggestion
				this.objectNodeFields.get(key) ?? fail("missing field"),
```